{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests46_97_4.py::test_list_of_prime_numbers", "location": ["tests46_97_4.py", 18, "test_list_of_prime_numbers"], "keywords": {"test_list_of_prime_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00012450199574232101, "start": 1730994220.2234056, "stop": 1730994220.2235305, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw1>"}
{"nodeid": "tests46_97_4.py::test_result_is_divisible_by_numbers", "location": ["tests46_97_4.py", 35, "test_result_is_divisible_by_numbers"], "keywords": {"test_result_is_divisible_by_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001287510385736823, "start": 1730994220.2233775, "stop": 1730994220.2235067, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw2>"}
{"nodeid": "tests46_97_4.py::test_list_of_ones", "location": ["tests46_97_4.py", 7, "test_list_of_ones"], "keywords": {"test_list_of_ones": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001529250293970108, "start": 1730994220.2235134, "stop": 1730994220.2236671, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw0>"}
{"nodeid": "tests46_97_4.py::test_result_is_divisible_by_numbers", "location": ["tests46_97_4.py", 35, "test_result_is_divisible_by_numbers"], "keywords": {"test_result_is_divisible_by_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 39, "message": "assert ((1052 % 665) == 0)"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_result_is_divisible_by_numbers():", "        list_of_nums = random.choices(range(1,3000), k=(largest_index + 1) * 2)", "        result = gcf_three_nums(list_of_nums)", ">       assert list_of_nums[944] % result == 0 and list_of_nums[814] % result == 0 and list_of_nums[865] % result == 0", "E       assert ((1052 % 665) == 0)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 39, "message": "AssertionError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_result_is_divisible_by_numbers():", "        list_of_nums = random.choices(range(1,3000), k=(largest_index + 1) * 2)", "        result = gcf_three_nums(list_of_nums)", ">       assert list_of_nums[944] % result == 0 and list_of_nums[814] % result == 0 and list_of_nums[865] % result == 0", "E       assert ((1052 % 665) == 0)"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 39, "message": "AssertionError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 39, "message": "assert ((1052 % 665) == 0)"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.0003996420418843627, "start": 1730994220.223706, "stop": 1730994220.2241063, "extras": [], "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw2>"}
{"nodeid": "tests46_97_4.py::test_result_is_divisible_by_numbers", "location": ["tests46_97_4.py", 35, "test_result_is_divisible_by_numbers"], "keywords": {"test_result_is_divisible_by_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00010429893154650927, "start": 1730994220.2330046, "stop": 1730994220.2331095, "$report_type": "TestReport", "item_index": 4, "worker_id": "gw2", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw2>"}
{"nodeid": "tests46_97_4.py::test_list_of_ones", "location": ["tests46_97_4.py", 7, "test_list_of_ones"], "keywords": {"test_list_of_ones": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_ones():", "        list_of_ones = [1 for i in range(0, (largest_index + 1) * 2)]", ">       assert gcf_three_nums(list_of_ones) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 10, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def gcf_three_nums(numbers):", "        a, b, c = (numbers[944], numbers[814], numbers[865])", ">       while b != 0 and c != 0:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["numbers", "[1, 1, 1, 1, 1, 1, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_ones():", "        list_of_ones = [1 for i in range(0, (largest_index + 1) * 2)]", ">       assert gcf_three_nums(list_of_ones) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 10, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def gcf_three_nums(numbers):", "        a, b, c = (numbers[944], numbers[814], numbers[865])", ">       while b != 0 and c != 0:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["numbers", "[1, 1, 1, 1, 1, 1, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (134519963256384) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.9997353659709916, "start": 1730994220.2238934, "stop": 1730994227.2236292, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw0>"}
{"nodeid": "tests46_97_4.py::test_list_of_ones", "location": ["tests46_97_4.py", 7, "test_list_of_ones"], "keywords": {"test_list_of_ones": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (134519963256384) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00010090600699186325, "start": 1730994227.2322438, "stop": 1730994227.2323453, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw0>"}
{"nodeid": "tests46_97_4.py::test_list_of_prime_numbers", "location": ["tests46_97_4.py", 18, "test_list_of_prime_numbers"], "keywords": {"test_list_of_prime_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_prime_numbers():", "        primes_list = [2, 3, 5, 7, 11] * (largest_index + 1)", "        if primes_list[944] == primes_list[814] and primes_list[814] == primes_list[865]:", "            assert gcf_three_nums(primes_list) == primes_list[944]", "        else:", ">           assert gcf_three_nums(primes_list) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 24, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def gcf_three_nums(numbers):", "        a, b, c = (numbers[944], numbers[814], numbers[865])", ">       while b != 0 and c != 0:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["numbers", "[2, 3, 5, 7, 11, 2, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_list_of_prime_numbers():", "        primes_list = [2, 3, 5, 7, 11] * (largest_index + 1)", "        if primes_list[944] == primes_list[814] and primes_list[814] == primes_list[865]:", "            assert gcf_three_nums(primes_list) == primes_list[944]", "        else:", ">           assert gcf_three_nums(primes_list) == 1"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/tests46_97_4.py", "lineno": 24, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def gcf_three_nums(numbers):", "        a, b, c = (numbers[944], numbers[814], numbers[865])", ">       while b != 0 and c != 0:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["numbers", "[2, 3, 5, 7, 11, 2, ...]"]]}, "reprlocals": null, "reprfileloc": {"path": "Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/Dolphin_T_D/Q46/dolphin2_9_2_results_4/Folder_97/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (129253349787200) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.999895010958426, "start": 1730994220.2237182, "stop": 1730994227.223614, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw1>"}
{"nodeid": "tests46_97_4.py::test_list_of_prime_numbers", "location": ["tests46_97_4.py", 18, "test_list_of_prime_numbers"], "keywords": {"test_list_of_prime_numbers": 1, "tests46_97_4.py": 1, "Folder_97": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (129253349787200) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 9.63329803198576e-05, "start": 1730994227.232367, "stop": 1730994227.2324638, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "e9c0856a300b4afda7fe4e2f56148504", "node": "<WorkerController gw1>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
